ckanext/gobar_theme/helpers.py

```diff
@@ -572,3 +576,22 @@ def search_for_value_in_config_file(field):
         return value.replace(field, '')[1:]
     except:
         return ''
+
+
+def search_for_cron_jobs_and_remove(comment_to_search_for):
+    # Buscamos y eliminamos los cron jobs que contengan el comment especificado por parámetro
+    if comment_to_search_for:
+        cron = CronTab(get_current_terminal_username())
+        jobs_with_specified_comment = cron.find_comment(comment_to_search_for)
+        cron.remove(jobs_with_specified_comment)
```

Comment:
Casi seguro que find_comment devuelve un iterable, y tenes que llamar cron.remove() por cada uno en un for loop.

Reply:
```
        for job in cron.find_comment(job_id):
            job.delete()
```

Reply:
Solamente debería haber un cron job con ese comment. Si hay más de uno, es porque alguien metió mano directamente con `crontab -e`, cosa que no debería ocurrir.
Si lo decís por el hecho de cómo funciona, ya testeé que `cron.find_comment()` encuentra lo que necesito borrar, y `cron.remove()` lo hace exitosamente.